src/screens/Account.jsx

```diff
@@ -1,23 +1,35 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect, useState, useContext } from "react";
 
 import CandidateForm from "../components/CandidateForm";
 import PageTitle from "../components/PageTitle";
 import Loader from "../components/Loader";
-import { EDIT_CANDIDATE_FIELDS } from "../mocks";
+import { AppContext } from "../Context";
+import { editAccount } from "../utils/request";
 
 const Account = () => {
   // 1. Subscribe to AppContext data
-  const [fields, setFields] = useState(EDIT_CANDIDATE_FIELDS);
+  const { user } = useContext(AppContext);
+  const [fields, setFields] = useState(user.personal);
+  const [loader, setLoader] = useState(true)
```

Comment:
Try naming your boolean values with "is", for example in this case, it would be "isLoading".